int32 mTagId
float32 xTagCenterPixel
float32 yTagCenterPixel
float32 closenessMetric